terraso_backend/apps/graphql/schema/users.py

```diff
@@ -35,19 +36,30 @@
 logger = structlog.get_logger(__name__)
 
 
-class UserNode(DjangoObjectType):
-    id = graphene.ID(source="pk", required=True)
+class UserFilter(FilterSet):
+    user__in_project = CharFilter(method="filter_user_in_project")
```

Comment:
nit: i think it'd be more idiomatic/consistent with the other filter names if the filter field was just named `project` rather than `user__in_project`